docs/Developers/Newcomers_Guide.rst

```diff
@@ -338,14 +338,14 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
```

Comment:
I think you should change commit message, it's the same as of the merged commit :+1: